riff-raff/public/docs/howto/fix-a-failed-deploy.md

```diff
@@ -1,5 +1,9 @@
 # Fixing a failed deploy
 
+> [!NOTE]
+> This guide is only necessary when using the legacy `autoscaling` deployment type.
```

Comment:
It might be worth linking out to more details here, just in case people are unsure what they're currently using?

```suggestion
> This guide is only necessary when using the legacy [`autoscaling` deployment type](https://riffraff.gutools.co.uk/docs/magenta-lib/types#autoscaling).
```